net/ntopng/pkg-descr

```diff
@@ -8,6 +8,12 @@ and on Windows as well.
 Plugin Changelog
 ================
 
+1.2 r2
```

Comment:
It's better to make this 1.3, revision is only for fixes et al

Reply:
done.